src/browser/filestorage.js

```diff
@@ -99,10 +99,11 @@ ServerFileStorageWrapper.prototype.load_from_server = function(sha256sum)
 {
     return new Promise((resolve, reject) =>
     {
-        v86util.load_file(this.baseurl + sha256sum, { done: buffer =>
+        v86util.load_file(this.baseurl + sha256sum, { done: async buffer =>
```

Comment:
Could you change v86util.load_file so that we can get rid of the Promise constructor here and make this function async too?